db/migrate/20200304012055_add_updated_alignment_config.rb

```diff
@@ -0,0 +1,20 @@
+class AddUpdatedAlignmentConfig < ActiveRecord::Migration[5.1]
+  NEW_DATE = "2020-02-10".freeze
+
+  def up
+    unless AlignmentConfig.find_by(name: NEW_DATE)
+      ENV["NCBI_DATE"] = NEW_DATE
+      ENV["LINEAGE_VERSION"] = (TaxonLineage.maximum('version_end') + 1).to_s
```

Comment:
I don't think the test environment has any `TaxonLineage` entries (at least mine doesn't), hence the Travis problems

Reply:
Thanks for finding that!